tests/phpunit/tests/functions/wpTriggerError.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * Test cases for the `wp_trigger_error()` function.
+ *
+ * @package WordPress\UnitTests
+ *
+ * @since 6.4.0
+ *
+ * @group functions.php
+ * @covers ::wp_trigger_error
+ */
+class Tests_Functions_WpTriggerError extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data_should_trigger_error
```

Comment:
Could you please add `@ticket` number in all the tests.

Reply:
Done in commit https://github.com/WordPress/wordpress-develop/pull/5144/commits/44fea6406f505979acfe94a2e0943691be3320e4 ✅